packages/data-service/src/data-service.ts

```diff
@@ -505,6 +620,16 @@ class DataService extends EventEmitter {
     );
   }
 
+  databaseStats(name: string, callback: Callback<Document>): void {
```

Comment:
can't we just make the `_databaseStats` public?

Reply:
Done in d67891d75cfb28248fd4e821971d89c1dbe07902